### ISSUE TITLE
Changing cell dimensions on netVisual_bubble?

### ISSUE THREAD

**Issue by stonejk** [sqjin/CellChat#292]
Hey, great package, it's been very helpful!

I'm comparing the interactions in different datasets. I've saved the netVisual_bubble plots for each data set and intend to arrange them vertically, but keep having issues with the graphs being different sizes. Number of cell types are the same in both sets. Is there a way to manually alter either the height or width of the "cells" within the bubble plot to coerce them to the same size?  Thanks!

**Comment by happylearning423** [comment]
@stonejk Have you considered generating the bubble plot using the merged CellChat object instead of doing it separately?

**Comment by stonejk** [comment]
Yes, thank you @happylearning423, that's what I eventually ended up doing, pulling the data out of the CellChat object and making the plot separately. I was just hoping for an internal solution.
